### PR TITLE
6 view all comments by article

### PR DESCRIPTION
**Pull request by MFStevenson**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,18 +1,33 @@
...
-import ArticleStateWrapper from "./components/ArticleStateWrapper";
 import HomePage from "./pages/HomePage";
 import TopicsPage from "./pages/TopicsPage";
+import SingleArticlePage from "./pages/SingleArticlePage";
+import Navbar from "./components/Navbar";
+import { useState } from "react";
+import CommentsPage from "./pages/CommentsPage";
 
 function App() {
+  const [articles, setArticles] = useState([]);
```
Comment (popatre) [inline src/App.jsx:13]: Cant see anything else thats using this state - so probably doesnt need to be in app

File: src/components/Articles.jsx [hunk 2]
```diff
@@ -1,10 +1,36 @@
...
-    console.log(articles)
+  const [input, setInput] = useState(0);
+  const navigate = useNavigate();
+
+  const updateInput = (e) => {
+    setInput(e.target.value);
+  };
+
+  const handleSubmit = (e) => {
+    console.log(e);
```
Comment (popatre) [inline src/components/Articles.jsx]: remove log

File: src/components/Articles.jsx [hunk 3]
```diff
@@ -1,10 +1,36 @@
...
 import ArticleCard from "./ArticleCard";
+import { useNavigate } from "react-router-dom";
+import { useState } from "react";
 
 const Articles = ({ articles }) => {
-    console.log(articles)
+  const [input, setInput] = useState(0);
+  const navigate = useNavigate();
+
+  const updateInput = (e) => {
```
Comment (popatre) [inline src/components/Articles.jsx]: Would a user know the id of the article they want?

File: src/pages/CommentsPage.jsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  };
+
+  if (isLoading) return <Loading />;
+
+  return (
+    <div id="comments-page">
+      <h2>Comments</h2>
+      <ul>
+        {comments.map((comment) => {
+          console.log(comment);
```
Comment (popatre) [inline src/pages/CommentsPage.jsx]: remove log

**Review by popatre — CHANGES_REQUESTED** [review]
Nice work so far. Really like the loading spinner

Just one thing please:
- [ ] You should add some feedback for when an article has no comments

**Review by popatre — APPROVED** [review]
Nice one - much better 👍

**Review by nc-modou — APPROVED** [review]
Great implementation of the comments, and the 'article has no comments' message! Merge away.

**Pull request merged**